index.html

```diff
@@ -427,11 +427,11 @@ <h3>Basic requirements</h3>
 
 
 	<div class="req" id="dir_paragraphs">
-	<p class="advisement">It must be possible to indicate base direction for each individual paragraph-level item of natural language text that will be read by someone.</p>
+	<p class="advisement">It must be possible to indicate base direction for each individual paragraph-level item of <a>natural language</a> text that will be read by someone.</p>
```

Comment:
@r12a I'd like to remove the awkward ".. that will be read by someone", perhaps by changing 'natural language' here to 'textual content'. Thoughts?